include/hip/hcc_detail/hip_runtime_api.h

```diff
@@ -2974,6 +2975,7 @@ hipError_t hipOccupancyMaxActiveBlocksPerMultiprocessorWithFlags(
 hipError_t hipExtLaunchMultiKernelMultiDevice(hipLaunchParams* launchParamsList,
```

Comment:
So with this change, hipExtLaunchMultiKernelMultiDevice - as a C API won't work?

Reply:
Is that what  `#if defined(__clang__) && defined(__HIP__)`  does? 
Also, the #if cpp pattern in other places seem to be different. For example, https://github.com/ROCm-Developer-Tools/HIP/blob/master/include/hip/hip_common.h#L29. Should there be an `#elif` check for `__HCC__` also?

Reply:
@srinivamd  The "hip_runtime_api.h" header is also used by HIP-CLANG and the current code makes sure that the definition of hipExtLaunchMultiKernelMultiDevice is untouched for HIP-CLANG. 
There is no #elf __HCC__ required here because the hipExtLaunchMultiKernelMultiDevice is defined in "functional_grid_launch.h" header for HIP-HCC and this header is not used in HIP-CLANG.

Reply:
@gargrahul yes, this won't have an C API on HIP/HCC.

Reply:
@srinivamd that guard means the code path is reserved for the hip-clang implementation